react/react1/week3/vite-project/src/todo-list.jsx

```diff
@@ -0,0 +1,42 @@
+import ListItem from "./todo-list-item";
+import "./index.css";
+import { useState , useEffect} from "react";
+
+const List = ({ todosList }) => {
+  debugger;
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => setTodos(todosList), [todosList]);
+
+  const onDelete = (id) => {
+    setTodos(todos.filter((item) => item.id !== id));
+  };
+
+  const handleSearch = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  const filteredList = todos.filter((todo) => {
+    return todo.title.toLowerCase().includes(searchTerm.toLowerCase());
+  });
+
+  return (
+    <div className="todo-list">
+      <input
```

Comment:
Give it a `value={searchTerm}` so that you can change the search value from your component if needed